sp800-63d/vot_mapping.md

```diff
@@ -1,54 +1,82 @@
-<a name="sec1"></a>
+<a name="vectors-of-trust-mapping"></a>
 
 <div class="breaker"></div>
 
-## VOT Overview
+## Mapping Assurance Levels to Vectors of Trust
+_This section is normative._
 
-VOT is comprised of components, and components contain values.
+Vectors of Trust (VoT) is a standard for expressing information about an authentication transaction from an IdP to an RP. This information is split into several orthogonal component categories, much in the same way that the assurances defined in 800-63 are split into different xAL categories.
```

Comment:
The assertion is passed around in the transaction, but we do mean the transaction here. We'll need to check -C to make sure we get the wording right.